src/jobs/veracode_pipeline_scan.yml

```diff
@@ -1,7 +1,9 @@
 ---
 description: >
   Veracode SAST pipeline scan
-executor: node
```

Comment:
can't see why these needed a `node` executor when just calling `java` and `wget`.  Surprised that `java` was indeed bundled with `node`.